three_numbers.py

```diff
@@ -0,0 +1,38 @@
+"""
+Given a sorted list of numbers s, and a target, return a list of 3
+numbers that add up to the target
+"""
+from typing import List
+
+def three_numbers(s: List[int], target: int) -> List[int]:
+    i  = 0
+    while i < len(s):
+        left = i + 1
+        right = len(s) - 1
+        while left < right:
+            total = s[i] + s[left] + s[right]
+            if total == target:
+                return [s[i], s[left], s[right]]
+            elif total < target:
+                left += 1
+            else:
+                right -= 1
+        i += 1
+
+"""
+Analysis:
+
+Time Complexity:
+    - for every number (n) in the list, we iterate over a subset (k) of the
+    list k/2 times. 
```

Comment:
The time complexity is correct but your analysis is a bit confusing. Why k/2 times?